### ISSUE TITLE
Style changes to docs for responsive resizing

### ISSUE THREAD

**Issue by anita-trimbur** [pulumi/theme#106]
## Overview

We'd like to increase the horizontal space for the labels on our code blocks in both the Registry and Docs, particularly for smaller screen sizes. Images below use mock tabs.

### Option 1: Column width adjustment
![Screen Shot 2022-04-13 at 1 03 1](https://user-images.githubusercontent.com/97129308/163241373-464c78fc-be78-4fa6-a61a-7ad56fa85a4f.png)

The right sidebar has been reduced in width, and the main content area has an increased width at the 1536 break point. At the 1280 breakpoint, we would flex the right sidebar vertically as we already do at the 1024 breakpoint. Screenshot modeled at 1280px. 

### Option 2: Flex the right sidebar vertically
![Group 17](https://user-images.githubusercontent.com/97129308/163241765-aaa9afb5-9697-437b-86a8-994348c5ea0e.png)

At the 1536 breakpoint, we could flex the right side bar vertically. This is more ideal in the Docs where the sidebar is content light, versus the Registry. Screenshot modeled at 1280px.

### Option 3: Reduce label size by using extension names
![Screen Shot 2022-04-13 at 2 06 48 PM](https://user-images.githubusercontent.com/97129308/163243121-6b5ffd96-2823-451b-ac53-bfdb46148c1c.png)

Abbreviate labels by using extension names. Suggested by @kimberleyamackenzie. Font-size has been reduced from 1em to 0.8em. Screenshot modeled at 1024px. Edit: This option is possible without any resizing if the maximum tab number is reduced from 7 to 6.

## Other directions

We can mock up these other options if needed, but my feeling is these directions are less ideal. 

- Tabs could scroll horizontally. This would allow all content to remain undisturbed, but it would obscure the last few tabs in the list.
- Tabs could take up two lines. This would also allow content to remain undisturbed, but it would take up double the vertical space we do now.
- Column padding could be reduced, which would shift content on the page, and potentially overcrowd text.
- Right sidebar could float and collapse over top the main content area. Potentially more eng time, and introduces a design pattern we don't use anywhere else.

**Comment by susanev** [comment]
thx for getting this opened! for the file extensions, we should use lowercase instead of uppercase

**Comment by anita-trimbur** [comment]
fixed!

**Comment by kimberleyamackenzie** [comment]
Sweet, thanks @anita-trimbur!  I'm gonna work on doing a quick hack of each of these options like we talked about, and will report back tomorrow with links and/or any big blockers  to any of these options on the eng side.  I'll start this afternoon, but I have some meetings, so not sure how much I'll get done - will update in tomorrow's team meeting.

**Comment by susanev** [comment]
@anita-trimbur can you add a comment with the direction that was decided?

**Comment by anita-trimbur** [comment]
All three options were reviewed with stakeholders on 4/15. 

Option 3 was ruled out because it presumes a user to be familiar with all language extensions to quickly read and understand each label. Between options 1 and 2, we discussed the trade-offs of less users seeing the right side bar versus potential overcrowding in the left navigation. 

We decided on option 2, concluding that the right side bar was lower-value and that we should consider a more thorough redesign in the future. For now, the right sidebar will flex vertically at the 1536 breakpoint, as it does now at the 1024 breakpoint.
